src/main/drivers/accgyro/accgyro_mpu.h

```diff
@@ -209,7 +211,9 @@ typedef enum {
     BMI_270_SPI,
     LSM6DSO_SPI,
     L3GD20_SPI,
-    LSM6DSV16X_SPI
+    LSM6DSV16X_SPI,
+    ICM_45605_SPI,
+    ICM_45686_SPI
```

Comment:
(next diff will look better this way):
```suggestion
    ICM_45686_SPI,
```